README.md

```diff
@@ -88,6 +99,27 @@ By default, all packages inside the repository are subject to coverage testing,
 The values in `only` and `except` are regexes for `grep -E`.
 Since only entire packages (not single source files) can be selected for coverage testing, the regexes have to match package names, not on file names.
 
+### `docker`
+
+```yaml
+docker:
```

Comment:
The section name `docker` sounds like we're running Docker as part of the Makefile or something. I would suggest to rename this section to `dockerfile:`.